templates/web/base/maps/openlayers.html

```diff
@@ -25,6 +25,9 @@
 [% IF include_key -%]
     data-key='[% c.config.BING_MAPS_API_KEY %]'
 [%- END -%]
+[% IF bodies_to_list -%]
+    data-bodies='[% body_names_to_list.join(',') %]'
```

Comment:
This will break if any body name includes either a comma or a single quote. Copy what the category listing does with its CSV thing, the same should work here I think?